documentation/src/main/wiki/FAQ.md

```diff
@@ -0,0 +1,144 @@
+[Bamboo](https://www.atlassian.com/software/bamboo)).
+
+### 1.4. I run AET for the first time, how the tool will know patterns?
+When running AET for the first time AET has no patterns. The first collected screenshots will be treated
+as patterns, so the first run will detect no changes. The same is true when a new test or url is added to
+already existing suite - new records will have patterns collected during the first suite run.
+
+
+---
+
+## 2. Environment and software
+
+### 2.1. Which browser is used by AET?
+AET currently uses Fire Fox 38.6.0 ESR as default browser. Information about current versions of components
+used by AET can be found in [[Third-party software used by system|SystemComponents#third-party-software-used-by-system]]
+section.
+
+
+### 2.2. Does AET use database for storing data?
+Yes, AET uses [MongoDB](https://www.mongodb.org/) where all data collected during AET suite processing 
+is stored.
+
+
+### 2.3. I have a problem with missing fonts (my page is not english). What should I do?
+You can add custom fonts to AET virtual machine. To do so, place desired fonts inside
+`usr/share/fonts` or `/usr/share/fonts/truetype` directories, AET's FireFox should use them.
+Other idea is to install fonts with e.g.:
+`yum groupinstall chinese-support`
+`yum groupinstall arabic-support`
+See more details in answers for [this AET question](https://github.com/Cognifide/aet/issues/61).
+
+---
+
+## 3. User interface and reports
+
+### 3.1 What browser should I use to browse AET reports?
+AET reports are dedicated and tested with Chrome browser, however they should also work fine with other 
+modern browsers like FireFox or Edge.
+
+### 3.2 I would like to compare pages between two environments/domains. How can I do that with the AET Tool?
+The AET System enables comparison of the same page (set of pages) between different environments (domains). 
+To do so, simply change domain parameter 
+(either by change in [[suite definition|DefiningSuite#suite]] or using [[aet run command|ClientApplication#parameters]]) 
+when running following tests. All data collected during those suite runs will be compared with the 
+same patterns (which are independent of domain attribute).
+
+### 3.3 How can I access pages that require authentication to enter?
+There are two possible solutions. For the Basic Authentication you may use [[Header Modifier|HeaderModifier]]
+with `Authorization` key set:
+```xml
+<header key="Authorization" value="Basic emVuT2FyZXVuOnozbkdAckQZbiE=" />
+```
+The other way is to use [[Login Modifier|LoginModifier]] that can pass simple login form.
+
+
+### 3.4 How can I simulate another user agent?
+Use [[Header Modifier|HeaderModifier]] to achieve that. Just add header modifier with key `'User-Agent'`
+ and requested value.
+
+
+### 3.5 What can I do with elements that render differently each time page is open and it occurs in changes detected each time?
```

Comment:
The past participle of _open_ is _opened_. Also, you're missing an article.

_each time **a** page is open**ed**_

Notice the difference between the two sentences:

_The door is open._ - in this case, _open_ is an adjective.

_The door was opened._ - in this case, _open_ is a verb. This sentence means that an unspecified party opened the door. This is similar to the context in which you used the word.